netbox/extras/forms/reports.py

```diff
@@ -25,20 +25,25 @@ class ReportForm(BootstrapMixin, forms.Form):
         help_text=_("Interval at which this report is re-run (in minutes)")
     )
 
+    def __init__(self, *args, scheduling_enabled=True, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        # Annotate the current system time for reference
+        now = local_now().strftime('%Y-%m-%d %H:%M:%S')
```

Comment:
maybe put a comment here to check for localization "TODO: LOCALIZATION" or such, not sure if this is good practice when we do localization